manuscript/developing/02_development_server.md

```diff
@@ -132,15 +142,15 @@ T> If you want even better output, consider [error-overlay-webpack-plugin](https
 
 ## Accessing development server from the network
 
-It's possible to customize host and port settings through the environment in the setup (i.e., `export PORT=3000` on Unix or `SET PORT=3000` on Windows). The default settings are enough on most platforms.
+To access your development server from the network, you need to figure out the IP address of your machine. For example, using `ifconfig | grep inet` on Unix, or `ipconfig` on Windows. An npm package, such as [node-ip](https://www.npmjs.com/package/node-ip), come in handy as well.
 
-To access your server, you need to figure out the ip of your machine. On Unix, this can be achieved using `ifconfig | grep inet`. On Windows, `ipconfig` can be utilized. An npm package, such as [node-ip](https://www.npmjs.com/package/node-ip) come in handy as well. Especially on Windows, you need to set your `HOST` to match your ip to make it accessible.
+On Windows you need to set your `HOST` to match your IP to make it accessible. **AS: how?**
```

Comment:
Let's add `HOST=<ip goes here> npm start` kind of tip here. The config should be able to pick it up.